core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -547,6 +547,10 @@ class TeamPolicy
 
   typedef TeamPolicy execution_policy;
 
+  const typename traits::execution_space& space() const {
```

Comment:
why do I need this? isn't that anyway coming from the base class?

Reply:
Yes, we can drop this.